splunk_add_on_ucc_framework/commands/openapi_generator/ucc_to_oas.py

```diff
@@ -106,6 +106,17 @@ def __get_schema_object(
         ):
             continue
         if schema_object.properties is not None:
+            if entity.field == "oauth":
+                # check for oauth as basic authentication is also mentioned in oauth
+                if "basic" in entity.options.auth_type:
```

Comment:
Why `in`? Looks like it supports only 2 options (https://github.com/splunk/addonfactory-ucc-generator/blob/4aa97bd9d278aff8ca6d818ab8608c96f347954a/splunk_add_on_ucc_framework/schema/schema.json#L1667-L1683).

Reply:
`in` because when using the oauth feature, the auth_type could have `basic`, `oauth` or `basic, oauth`, hence we want to generate only when basic is present in this list.

Reply:
you are right, thanks the explanation!